freebasics/templates/500.html

```diff
@@ -1,21 +1,14 @@
 
+
+{% extends "error_page_base.html" %}
```

Comment:
Aaah, good point. I've made it standalone -- and ugly.

![screen shot 2016-04-22 at 11 01 31](https://cloud.githubusercontent.com/assets/44849/14736942/f0b573f4-0879-11e6-8c09-27353f05f571.png)